### PR TITLE
watch

### PR DESCRIPTION
**Pull request by kkiirill**
[DEMO LINK](https://kkiirill.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  @keyframes spin {
+    0% {
+      transform: rotate(-180deg);
+    }
+    100% {
+      transform: rotate(180deg);
+    }
+  }
+}
```
Comment (Olexandr6) [inline src/styles/main.scss:63]: Create a separate file per each BEM block

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  width: 300px;
+  height: 300px;
+  border: 5px solid rosybrown;
+  border-radius: 50%;
+  left: 50%;
+  top: 50%;
+  background: #fefbf2;
+  transform: translate(-50%, -50%);
+  box-shadow: inset 0 0 10px black;
+  &__minute {
```
Comment (Olexandr6) [inline src/styles/main.scss:29]:
```suggestion
  box-shadow: inset 0 0 10px black;
  
  &__minute {
```

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Almost done👍, check comments👇

File: src/styles/watch.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  height: 300px;
+  border: 5px solid rosybrown;
+  border-radius: 50%;
+  left: 50%;
+  top: 50%;
+  background: #fefbf2;
+  transform: translate(-50%, -50%);
+  box-shadow: inset 0 0 10px black;
+
+  &__minute {
```
Comment (Ihor-Karpyn) [inline src/styles/watch.scss:15]:
```suggestion
  &__second {
```

File: src/styles/watch.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    width: 10px;
+    background: rgb(237, 63, 63);
+    top: 50%;
+    @include circle;
+    border-radius: 5px;
+    animation: spin 60s linear infinite;
+    transform-origin: top;
+  }
+
+  &__second {
```
Comment (Ihor-Karpyn) [inline src/styles/watch.scss:28]:
```suggestion
  &__minute {
```

File: package.json [hunk 5]
```diff
@@ -23,7 +23,7 @@
     "@mate-academy/bemlint": "^0.1.1",
     "@mate-academy/eslint-config": "*",
     "@mate-academy/linthtml-config": "0.0.2",
-    "@mate-academy/scripts": "^0.9.1",
+    "@mate-academy/scripts": "^0.9.14",
```
Comment (Ihor-Karpyn) [inline package.json:26]: don't add this

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**